_policies/regressions.md

```diff
@@ -1,8 +1,39 @@
 ---
-title: Regressions
+title: How to Mark Regressions
 layout: policy
-published: false
+published: true
 description: Marking-up regressions
 ---
 
-Page under development.
\ No newline at end of file
+# What is a regression?
+
+A regression is a bug (in our scheme a `defect`) introduced by a change set. 
```

Comment:
changeset (see https://en.wikipedia.org/wiki/Changeset)

Reply:
Adding definition and fixing spelling.